models/index.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+
+const fs        = require("fs");
+const path      = require("path");
+const Sequelize = require("sequelize");
+const env       = process.env.NODE_ENV || "development";
```

Comment:
`require('./environment.js').get('node_env');` since we have an `environment.js` config file. If you want it to be set from command line, probably add logic to that file similar to `LOG_LEVEL`